src/fr/bluesolo/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'Blue Solo'
+    extClass = '.BlueSolo'
+    themePkg = 'pizzareader'
+    baseUrl = 'https://bluesolo.org/'
```

Remove the slash from the end of baseUrl

```suggestion
    baseUrl = 'https://bluesolo.org'
```